core/src/main/java/org/kohsuke/stapler/RequestImpl.java

```diff
@@ -369,7 +369,7 @@ public BindInterceptor setBindListener(BindInterceptor bindListener) {
         return setBindInterceptor(bindListener);
     }
 
-    public BindInterceptor setBindInterceptpr(BindInterceptor bindListener) {
+    public BindInterceptor setBindInterceptor(BindInterceptor bindListener) {
```

Comment:
This is an API break. I can provide a separate PR that includes a deprecation annotation if requested...